### ISSUE TITLE
Visualize Update progress

### ISSUE THREAD

**Issue by dploeger** [keeweb/keeweb#1754]
**Is your feature request related to a problem? Please describe.**
I used the internal updater to update to a minor version. It said "Downloading update" and that took a while. Because nothing happened, I quitted Keeweb and tried again. This time, the updater said "Download error". Upon restarting the update, it worked. I guess the main problem was, that it was actually *downloading* the update (and that took a while), but didn't visualize it properly that progress that lead me to believe, it got stuck.

**Describe the solution you'd like**
When showing "Downloading update", visualize the download progress by a progress bar or percent numbers and a download rate.

**Describe alternatives you've considered**
An alternative would be to show a popup for "Downloading update". That would it more clear, that it is actually doing something now.

**Comment by antelle** [comment]
I agree. The internal updater was designed to work with small (≈1..2Mb) updates, this didn't take much time to load. Now it updates the whole app, which is 100..200Mb, this takes a considerable amount of time and we should add the progress bar.
